client/components/Header.js

```diff
@@ -1,3 +1,4 @@
+import jquery from 'jquery'
```

Comment:
Me neither! But since the normal bootstrap library is used and not a react library such as reactstrap, this is currently the easiest and shortest solution.